documentation/how-to/prepare-for-production.md

```diff
@@ -0,0 +1,193 @@
+
+# Prepare for Production
+
+## Performance Optimisation
+
+### Enable Asynchronous Querying
+To enable asynchronous querying, you need to deploy Charmed Superset Workers. These workers handle long-running queries asynchronously, allowing the UI to remain responsive. You can deploy additional Superset workers to offload and process background queries. Deploy Charmed Superset workers to the same model as the server with:
+
+```
+# Deploy Charmed Superset with worker functionality
+juju deploy superset-k8s --config charm-function=worker superset-k8s-worker
+
+# Relate with Charmed PostgreSQL and Charmed Redis
+juju relate superset-k8s-worker postgresql-k8s
+juju relate superset-k8s-worker redis-k8s
+```
+Within the UI you can enable Asynchronous Query Execution (`AGE`) at the database level.
+Edit the database and under the `Performance` tab check the `Asynchronous query execution`
+box.
+
+This is recommended for all production databases to relieve load on the UI.
```

Comment:
```suggestion
[note]
This is recommended for all production databases to relieve load on the UI.
[/note]
```
This is better presented as a note box. [note]...[/note] is Discourse nomenclature.